packages/teams-js/test/internal/utils.spec.ts

```diff
@@ -367,6 +368,105 @@ describe('utils', () => {
     });
```

Comment:
>}); [](http://example.com/codeflow?start=4&length=3)

Unless I'm missing something I think we don't have a unit test that validates a valid appId can be constructed successfully.